cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -37,6 +37,9 @@ const (
 
 // CreateLocalEtcdStaticPodManifestFile will write local etcd static pod manifest file.
 func CreateLocalEtcdStaticPodManifestFile(manifestDir string, cfg *kubeadmapi.InitConfiguration) error {
+	if cfg.ClusterConfiguration.Etcd.External != nil {
```

Comment:
Small nit
Please use fmt.Errorf. it is the standard in kubeadm codebase

Reply:
I don't have an opinion here because I am missing context on how the project wants to make error reporting homogeneous.

Reply:
@neolit123 
I'm in favour of errors too, but if we have to switch we should do this across all the kubeadm codebase.
In the meantime fmt.Errorf is the standard to use